awx/conf/fields.py

```diff
@@ -96,6 +100,17 @@ def to_internal_value(self, data):
         self.fail('type_error', input_type=type(data))
 
 
+class StringListPathField(StringListField):
+
+    def __init__(self, **kwargs):
+        super(StringListPathField, self).__init__(**kwargs)
+        self.validators.append(lambda sp: list(map(validate_system_path, sp)))
+
+    def to_representation(self, value):
+        # Normalize each path value and filter potential duplicates
+        return super(StringListField, self).to_representation(list(set(map(os.path.normpath, value))))
```

Comment:
Same suggestion here, let's make this a list comprehension, or just a loop so it's easier to read.

Reply:
Good idea w/ `normpath` to catch duplicates w/ trailing `/`.

Reply:
I think we would want a stable ordering of the list, so maybe also sort it?